cluster/log-dump/log-dump.sh

```diff
@@ -40,6 +40,7 @@ fi
 readonly master_ssh_supported_providers="gce aws"
 readonly node_ssh_supported_providers="gce gke aws"
 readonly gcloud_supported_providers="gce gke"
+readonly detect_project_supported_providers="gce gke"
```

Comment:
gke is not valid in this list, there's no `cluster/gke` anymore. I don't think it matters in the later logic, since it *should* bypass the `-z "${use_custom_instance_list}"` logic as well, but it's hard for a casual reader to work that out.